providers/bnxt_re/verbs.c

```diff
@@ -150,7 +150,7 @@ int bnxt_re_alloc_page(struct ibv_context *ibvctx,
 	DECLARE_COMMAND_BUFFER(cmd,
 			       BNXT_RE_OBJECT_ALLOC_PAGE,
 			       BNXT_RE_METHOD_ALLOC_PAGE,
-			       4);
+			       5);
```

Comment:
Your patch looks correct to me. However there is a need to format it according to our patch rules (same as in the kernel). Please write commit message which describes the issue and add Fixes line which points to the commit which is fixed here.